src/Illuminate/Session/TokenMismatchException.php

```diff
@@ -6,5 +6,14 @@
 
 class TokenMismatchException extends Exception
 {
-    //
+    /**
+     * Create a new token mismatch exception.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        $this->message = __('Sorry, your session has expired. Please refresh and try again.');
```

Comment:
This is a `Foundation` helper and the `illuminate/session` package does not depend on that.